### ISSUE TITLE
Atem and CasparCG Client integration does not work

### ISSUE THREAD

**Issue by nicoaguero** [CasparCG/client#226]
I've never been able to use the function CasparCG Client with Television Studio Atem, when I set up the switcher, the program stops working. And unusable. The only way to do work again is to remove the program creates folders in C:\Users\(username)\.CasparCG

Anyone have any solution?

I've also tested on Mac, but the program is not connected to the switcher.

![caspar](https://cloud.githubusercontent.com/assets/23171964/19859495/2fe8d2a6-9f64-11e6-9321-68a55ecb89a7.png)

**Comment by dotarmin** [comment]
I noticed on the forum that other also has reported same or similar problems to this one.

**Comment by TracyPeterson** [comment]
I was one of the users that reported it.  On my current machine I just
tested and no problerm after adding the mixer during the session.

Atem software: 6.9
Desktop Video: 10.6.5
Client: 2.0.8

**Comment by dotarmin** [comment]
@TracyPeterson Thanks for the information, I will look into this. 
It would be really good all users with same issue could provide the version of Atem software & Client version.

/Armin

**Comment by nicoaguero** [comment]
Caspar CGClient version: 2.0.8
Atem software: 6.9

**Comment by edinc90** [comment]
Same problem here. 

Desktop Video 10.6.8 (also tried each version up to 10.7.1)
ATEM Control 6.9
CasparCG Client 2.0.8

I rolled back to Client 2.0.7 as a temporary fix. Oddly, the issue didn't show up until I got a new Decklink card (Quad 2) and updated Desktop Video to 10.6.8. I updated ATEM Control to 6.9 by itself with no problems.

**Comment by dotarmin** [comment]
@edinc90 interesting, thanks for the information.

**Comment by dotarmin** [comment]
I have tried to reproduce the bug without any success. I have tried all kind of combinations etc., I will continue to try to locate the issue.

**Comment by nicoaguero** [comment]
I have also tried, in different versions of Windows, as with different versions of the software of Atem, without obtaining results. The error he threw was the same. It is a great function for who integrates Atem and CasparCG, I hope that a solution is found.

**Comment by bencredible** [comment]
I'm able to replicate this across the board with an ATEM 2 M/E switcher that is running firmware 7.1.1 -- Not sure if you can do the same but it appears to be linked to qatemcontrol.dll

**Comment by nicoaguero** [comment]
I have not posted here but the only way to work with CasparCG Client and its utility for atem was with a pc with windows installed again, without installing anything Blackmagic at all. I work very well, I work together Atem / Caspar to carry out the realization of more automated TV programs.

**Comment by dedicatedbroadcastsolutions** [comment]
Fixed with pull request #251 for both branches on here.  Someone needs to update the qatemcontrol libraries for linux and mac, but I got it fixed for windows.

Here are a couple of builds I put together for windows.  These were forked from the latest commits on each branch.

Here is a build with the included fix.
Master Branch 2.0.8 (current)
https://goo.gl/837uVn

Next Branch 2.0.9
https://goo.gl/yQteYB

These links will probably go dead once there are official builds including these changes.
